repos/system_upgrade/el7toel8/models/repositoriessetuptasks.py

```diff
@@ -0,0 +1,8 @@
+from leapp.models import Model, fields
+from leapp.topics import SystemFactsTopic
+
+
+class RepositoriesSetupTasks(Model):
+    topic = SystemFactsTopic
```

Comment:
I think we should consider using a specific topic just for all those RPM and Repository related things. Just a note

Reply:
Also something for a future PR?

Reply:
yes for future PR